lib/active_fedora/association_hash.rb

```diff
@@ -0,0 +1,89 @@
+require 'forwardable'
+
+module ActiveFedora
+  class AssociationHash
+    extend Forwardable
```

Comment:
No idea. As far as I can tell it doesn't use it - was brought over from FilesHash. I could remove it, but I'm not sure what the project's policy on refactors + feature requests in one PR is.